ui/guest/src/components/GuestProxy.js

```diff
@@ -277,19 +284,39 @@ export function GuestProxy(props) {
             $clone.insertBefore($siblings.eq(targetIndex));
           }
 
-          forEach(
-            $daddy.children(),
-            (el, i) => {
+          registerDeregisterElements(Array.from($daddy.children()), 'insert', targetIndex);
+
+          break;
+        }
+        case INSERT_INSTANCE_OPERATION: {
+          const { modelId, fieldId, targetIndex, instance } = op.args;
+
+          let $clone = $(`[data-craftercms-field-id="${fieldId}"][data-craftercms-index]:first`).clone();
+
+          if ($clone.length) {
+            Object.keys(instance).forEach((field) => {
+              if (!field.endsWith('_o') && field !== 'craftercms') {
+                //cleaning CDATA and escapedhtmltags
+                let value = instance[field].replace("<![CDATA[", "").replace("]]>", "");
+                value = stripEscapedHtmlTags(value);
+                $clone.find(`[data-craftercms-field-id="${field}"]`).html(value);
```

Comment:
You should not be getting CDATA or serialization details at this point. When parsing the content, the service should parse to "app format". Check service.

Reply:
for CDATA that is true, but for htmlTags there are needed because, we are retrieving the text inside of a RTE